backend/src/main/backend/Table.java

```diff
@@ -0,0 +1,100 @@
+import java.sql.Connection;
+import java.sql.PreparedStatement;
+import java.sql.SQLException;
+import java.util.ArrayList;
+
+public class Table {
+    private String name;
+
+    private Connection mConnection;
```

Comment:
I think the only member variable in this class that's used is `name` - do these others need to be here?

Reply:
mConnection is not used but I use the others. They are declared here then initialized as prepared statements in each method before their values are then set and the prepared statement is executed. 

for example, lines such as:
```
db.mDropTable = db.mConnection.prepareStatement(statement);
db.mDropTable.execute();
```
in the `drop` method make use of the mDropTable variable. My gut says this is a redundant way of going about executing the prepared statement. perhaps i could just write the above lines as
```
db.mConnection.prepareStatement(statement).execute();
```
and eliminate all those member variables. what do you think?